app/src/main/java/com/hyperion/ths/marvel_03/utils/Constant.java

```diff
@@ -0,0 +1,42 @@
+package com.hyperion.ths.marvel_03.utils;
+
+import java.math.BigInteger;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * Created by ths on 27/05/2017.
+ */
+
+public final class Constant {
+    private Constant() {
+
+    }
+
+    public static final String END_POINT_URL = "https://gateway.marvel.com";
+    public static final String PUBLIC_KEY = "4582ff7d7509676253b6bc474a0020fb";
+    public static final String PRIVATE_KEY = "4acceb2f9d2ab768c604c81a5f371b2e114b554e";
+    public static final long TIMESTAMP = 131710;
+    public static final int CONNECTION_TIMEOUT = 60;
+
+    public static String getHashKey() {
+        String hashKey = getMD5EncryptedString(TIMESTAMP + PRIVATE_KEY + PUBLIC_KEY);
+        return hashKey;
+    }
+
+    public static String getMD5EncryptedString(String encTarget) {
+        MessageDigest mdEnc = null;
+        try {
+            mdEnc = MessageDigest.getInstance("MD5");
+        } catch (NoSuchAlgorithmException e) {
+            System.out.println("Exception while encrypting to md5");
```

Comment:
ERROR: [SystemPrintln] System.out.println is used
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/logging-java.html#SystemPrintln